packages/input/src/ux-input.ts

```diff
@@ -43,8 +46,17 @@ export class UxInput implements UxInputComponent {
   public value: any;
   public textbox: HTMLInputElement;
 
-  constructor(private element: UxInputElement, public styleEngine: StyleEngine) {
+  constructor(private element: UxInputElement, public styleEngine: StyleEngine, private defaultConfiguration: UxDefaultInputConfiguration) {
```

Comment:
if the configuration is not used anywhere else after the constructor, let's not assign it to the ux input

Reply:
done